{"name": "test_login_error1[data1]", "status": "passed", "steps": [{"name": "获取返回的登录页面", "status": "passed", "start": 1570277667367, "stop": 1570277667367}, {"name": "输入错误的用户名与密码，点击登录", "status": "passed", "start": 1570277667367, "stop": 1570277667679}, {"name": "获取登录失败后错误的提示", "status": "passed", "start": 1570277667679, "stop": 1570277667686}, {"name": "断言", "status": "passed", "start": 1570277667686, "stop": 1570277667697}], "parameters": [{"name": "data", "value": "{'user': '135', 'pwd': '', 'expected': '请输入正确的手机号'}"}], "start": 1570277667367, "stop": 1570277667697, "uuid": "dcf930a6-e9f7-4266-994d-ab366ebfdc36", "historyId": "33483599eb7cf5b997550e7c506db57f", "testCaseId": "fc47bc21e513f1c18bc4b9d1e5ef13ea", "fullName": "test_cases.test_login.TestLogin#test_login_error1", "labels": [{"name": "story", "value": "登录失败用例"}, {"name": "feature", "value": "测试登录功能"}, {"name": "tag", "value": "success"}, {"name": "parentSuite", "value": "test_cases"}, {"name": "suite", "value": "test_login"}, {"name": "subSuite", "value": "TestLogin"}, {"name": "host", "value": "DESKTOP-I3J3MGM"}, {"name": "thread", "value": "9172-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "test_cases.test_login"}]}